src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -1116,3 +1118,25 @@ def pd_read_json(file) -> "DataFrame":
 
     with read_open(file) as f:
         return pd.read_json(f, orient="records", lines=True)
+
+
+def get_uuid_by_mac_id():
+    try:
+        node = str(uuid.getnode())
```

Comment:
what if the machine has two mac?

Reply:
If there are multiple MACS, this function will only obtain the first valid MAC ID

But if multiple virtual machines or containers share the same MAC, then these multiple virtual machines or containers will obtain the same MAC ID.

Reply:
According to the discussion, as there may be multiple MAC addresses, and MAC addresses may increase or decrease, always use the MAC address of the first network card to ensure that the MAC addresses obtained by the same machine are always consistent as much as possible.